drop-transfer/src/ws/server/v2.rs

```diff
@@ -189,8 +189,29 @@ impl<const PING: bool> HandlerLoop<'_, PING> {
             }) = self.jobs.remove(&file)
             {
                 if !task.is_finished() {
+                    let file_id = self
+                        .xfer
+                        .file_by_subpath(&file)
+                        .expect("File should be there sice we have a task registered")
+                        .id();
```

Comment:
```suggestion
                        .file_by_subpath(&file)
                        .expect("File should be there since we have a task registered")
                        .id();
```

Reply:
done